public/js/controllers/traffic_metrics_controller.js

```diff
@@ -49,7 +49,8 @@ export class TrafficMetricsController {
         let node = d3.select("#AT-units-bal");
         if (data.length) {
             let { balance } = data[0];
-            node.text(balance);
+            let [ currency, amount ] = balance.split(" ");
+            node.text(`${Number(amount).toLocaleString("en-KE", { style: "currency", currency })}`);
         } else {
             node.text("N/A");
```

Comment:
Drive-by comment for another PR: can this be "N/A" only if AfricasTalking isn't present in the project configuration? If it is, and there is no data, show an error here instead. This also means if we remove africastalking from the configuration this should switch back to "N/A"

Reply:
Noted